### DISCUSSION TITLE
Welcome to NeuralRecon Discussions!

### DISCUSSION THREAD

**Discussion by JiamingSuen** [zju3dv/NeuralRecon#1] [General]
## 👋 Welcome!
We’re using Discussions as a place to discuss all the questions related to the paper, in the hope of leaving the "Issues" section only to questions related to the implementation.

**Comment by cyrilzakka** [comment]
Hello, great paper but I'm curious about a few things. 

1) How does neural recon handle movement in the scene? Let's say a cat jumps into the scene - will it update the TSDF accordingly?
2) Does it explicitly need camera position for training? Or can it infer it from video?

**Reply by JiamingSuen** [reply]
Thanks for the questions!
1. Like most of the current 3D reconstruction systems, NeuralRecon cannot handle dynamic scenes for now. It would be an interesting research problem to add this ability to NeuralRecon.
2. Yes, we assume the camera poses are assumed known during training and inference. This assumption is easy to satisfy with ARKit/ARCore on modern smart phones, which is demonstrated in the incremental reconstruction on the project page.
